package.json

```diff
@@ -9,7 +9,7 @@
         "watch:app": "cd ./app && npm run build:watch:with-firebase",
         "watch:functions": "cd ./functions && npm run build:watch",
         "start:firebase": "firebase emulators:start --project demo-project-id --import=./emulator-seed-data",
-        "start": "concurrently \"npm run watch:app\" \"npm run watch:functions\" \"npm run start:firebase\""
+        "start": "concurrently -n watch:app,watch:functions,firebase \"npm run watch:app\" \"npm run watch:functions\" \"npm run start:firebase\""
```

Comment:
I'm not sure what this is doing now?

Reply:
Just added this so the console log streams have meaningful names rather than the index of the running process

Reply:
So instead of `[0], [1], [2]` it is `[watch:app], [watch:functions], [firebase]`

Reply:
Have updated the syntax to remove the `-n` flag and instead written all `npm run xxx` commands are written as `npm:xxx` and concurrently figures out to use `xxx` as the process name.